### PR TITLE
Ensure 2.5 browser tests is in the correct tab + change cypress test user name

### PR DESCRIPTION
**Pull request by DurkoMatko**
**fix:** 2.5 browser tests by adding data-testids to all user tabs (expired, deactivated) and ensure the collect tab is selected when applying actions on a checkbox

**change:** test beneficiary name (just a test to see whether it helps the 5.2 tests to pass...I was thinking that maybe the issue is that we're testing with user named "Smith"...who's all the way down in the table...who knows why cypress doesn't find him, he exists in the DOM...but maybe this will help...altho not ideal)

### CODE REVIEW

File: cypress/integration/2_5_UserManagement.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ describe('User management', () => {
         getDeactivateButton().click();
         getConfirmActionButton().click();
         getDeactivatedUsersTab().click();
+        getDeactivatedUsersTab().should('have.class', 'active')
```
Comment (DurkoMatko) [inline cypress/integration/2_5_UserManagement.js]: The issue was that cypress wasn't waiting till the deactivated page loaded and checked the checkbox in the "active" tab...then deactivated got loaded and suddenly there was no checkbox set to 'true'...it was a timing issue, that's why the test sometimes passed and sometimes failed..

**Comment by HaGuesto** [comment]
[Cypress test 2_5 adding testids](https://trello.com/c/byHwgVnG/397-cypress-test-25-adding-testids)

File: cypress/integration/5_2_Add_Beneficiary.js [hunk 2]
```diff
@@ -104,7 +105,7 @@ context("5_2_Add_Beneficiary_Test", () => {
         //check all the forms 
         FillForm(Test_firstname, Test_lastname, Test_case_id);
         ClickButtonWithText("Save and close");
-        cy.notificationWithTextIsVisible(Test_firstname + " " + Test_lastname + " was added");
+        //cy.notificationWithTextIsVisible(Test_firstname + " " + Test_lastname + " was added");
```
Comment (HaGuesto) [inline cypress/integration/5_2_Add_Beneficiary.js]: Why did you comment out this test?

Comment (DurkoMatko) [reply cypress/integration/5_2_Add_Beneficiary.js]: No idea :D was probably just trying something and forgotten to add it back :)

Comment (HaGuesto) [reply cypress/integration/5_2_Add_Beneficiary.js]: ok, could you add it back then?

Comment (DurkoMatko) [reply cypress/integration/5_2_Add_Beneficiary.js]: Done

**Pull request merged**